src/main/java/core/basesyntax/FruitService.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FruitService {
+    public static final Logger LOGGER = LoggerFactory.getLogger(FruitService.class);
```

Comment:
```suggestion
    public static final Logger logger = LoggerFactory.getLogger(FruitService.class);
```
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
fixed